Second training ISSUE4~6の修正

K-Hir-pk18 opened the pull request:
ISSUE4~6の修正

K-Hir-pk18 committed:
Issue4

formクラスのヘルパーメソッドに
HTTPメソッドを追加

K-Hir-pk18 committed:
issue5

日月のヘルパーメソッドの変更
非表示→表示

K-Hir-pk18 committed:
issue6

曜日の追加

TakumaKawashima commented:
レビュー依頼ありがとうございます！
こちらのプルリクエストにて、コンフリクトが起きていますので、まずはこちらを解消してください！
ブラウザエディタでコンフリクトを解消できますので、「Resolve conflicts」をクリックしましょう。

K-Hir-pk18 committed:
曜日の追加２

前回PR から微変更あり。
コンフリクト解消に進むため
再度アップ

K-Hir-pk18 committed:
Merge branch 'master' into second_training

yuevo commented on app/controllers/calendars_controller.rb:
```diff
@@ -34,7 +34,14 @@ def get_week
+        wday_num = wday_num -7
+      end
+
+      days = { :month => (@todays_date + x).month, :date => @todays_date.day + x, :plans => today_plans, :wday => "wdays[#{wday_num}]" }
```
```
days = { :month => (@todays_date + x).month, :date => @todays_date.day + x, :plans => today_plans, :wday => "wdays[#{wday_num}]" }
```
こちら、ロケットハッシュの記述になっております。修正しましょう！

```
"wdays[#{wday_num}]"
```
こちらはダブルクオーテーションで囲ってしまうと、文字列として認識されてしまいます。
こちらでは外しておきましょう！

K-Hir-pk18 replied:
https://gyazo.com/dc87c7b2837a3f22c534ddb4b7b7bc68

issue6がどうしても分かりません。
ハッシュの添字に変数を導入する方法がわかればできると思っているのですが…。

変数に導入しようとしたり、記号を変えてみたりしているのですが皆目見当がつきません。
ヒントだけでもいただけないでしょうか。

yuevo commented on app/views/calendars/index.html.erb:
```diff
@@ -1,4 +1,6 @@
-  <%= form_with model: @plan, url: calendars_path, class: 'form' do |f| %>
+<div>
```
```
<div>
```
こちらのdivタグは不要なので削除しましょう！
また、そうなると閉じタグも削除する必要があるので、そちらも修正しましょう！

K-Hir-pk18 replied:
修正しました！

yuevo reviewed (commented):
レビューのご依頼ありがとうございます！
２点コメントさせていただいました。

また、こちらのプルリクエストですが、実際の画面や機能ができているか確認させていただきたいので、キャプチャ動画をご提出いただけますか？
「Gyazo　使い方」とgoogleで検索し、ツールによって生成されたURLをこのPRに貼っていただければ幸いです。
その後、再度レビュー依頼フォームにてご提出をお願いします。
ご確認のほど、よろしくお願いいたします！

K-Hir-pk18 committed:
Issue4~6: Commentの対応

(calenders_controller.rb)
・ロケットハッシュの解消
・wdaysのダブルクオーテション解除

(index.html.erb)
・<div>タグの削除

yuevo commented:
レビューのご依頼ありがとうございます！
```
issue6がどうしても分かりません。
ハッシュの添字に変数を導入する方法がわかればできると思っているのですが…。

変数に導入しようとしたり、記号を変えてみたりしているのですが皆目見当がつきません。
ヒントだけでもいただけないでしょうか。
```
こちらに関してですが、申し訳ございませんがレビュワーからは技術的なヒントは致しかねます。
お手数ですが、コネクトにて通話リクエストかチャットにてメンターにお問い合わせしてただければと存じます。
何卒、よろしくお願いいたします。

K-Hir-pk18 committed:
issue6部分修正

・viewファイルの修正

K-Hir-pk18 commented:
承知いたしました。

shukiyamamoto commented:
レビュー依頼ありがとうございます！
恐れ入ります、こちらのプルリクエストですが、実際に機能ができているか確認させていただきたいので、キャプチャ動画をご提出いただけますでしょうか？

ツールによって生成されたURLをこのPRに貼っていただければ幸いです。
その後、再度レビュー依頼フォームにてご提出をお願いいたします。

K-Hir-pk18 commented:
近藤です。承知いたしました。
時間ギリギリのご対応まことにありがとうございます。
以下がGYAZOのキャプチャです。
よろしくお願いいたします。

https://gyazo.com/dff655fab7fb1231ff4251167a02e19a

yoshi-chiba commented:
レビュー依頼ありがとうございます。
LGTMとさせていただきます。実装お疲れさまでした。

マージ後、ローカルのmasterブランチに忘れずにプルしましょう。
その後、新しいブランチを作成し次の実装に取り組んでいただけますと幸いです。
引き続き頑張っていきましょう。
